modules/troubleshooting-cluster-upgrader.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * upgrade/upgrade-roxctl.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="troubleshooting_upgrader_{context}"]
+= Troubleshooting Cluster Upgrader
+
+[role="_abstract"]
+This section describes potential problems and their solutions when using legacy installation method for the secured cluster and enabling the automated updates.
+The errors described here can be found in the clusters view when the upgrader fails.
+
+== Upgrader is missing permissions
+
+.Symptom
+
+The following error is displayed in the cluster page:
+
+[quote]
```

Comment:
Use different formatting per [OpenShift guidelines:](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#system-messages)

[source,text]
----
Upgrader failed to execute PreflightStage of the roll-forward workflow: executing stage "Run preflight checks": preflight check "Kubernetes authorization" reported errors. This usually means that access is denied. Have you configured this Secured Cluster for automatically receiving upgrades?"
----

Reply:
Done in 50ee5ad655e7c208ca3f7755c9614a32fe3c14d4 (will be squashed)

Reply:
I followed the instructions but the preview looks strange - see the "- text" at the bottom. What could we do about that?
![Screenshot 2024-09-17 at 10 18 01](https://github.com/user-attachments/assets/8ddf63b8-54d3-4aa2-9e1a-f93a1722cc2e)

Reply:
Found it, I was using 4 underscores instead of 4 hyphens to mark the block.